helper/config.go

```diff
@@ -87,35 +88,37 @@ func init() {
 
 // Configuration represents heimdall config
 type Configuration struct {
-	MainRPCUrl  string `json:"mainRPCUrl"`  // RPC endpoint for main chain
-	MaticRPCUrl string `json:"maticRPCUrl"` // RPC endpoint for matic chain
+	EthRPCUrl        string `mapstructure:"eth_RPC_URL"`        // RPC endpoint for main chain
+	BorRPCUrl        string `mapstructure:"bor_RPC_URL"`        // RPC endpoint for bor chain
+	TendermintRPCUrl string `mapstructure:"tendermint_RPC_URL"` // tendemint node url
 
-	AmqpURL           string `json:"amqpURL"`           // amqp url
-	HeimdallServerURL string `json:"heimdallServerURL"` // heimdall server url
-	TendermintNodeURL string `json:"tendermintNodeURL"` // tendemint noed url
+	BorChainID string `mapstructure:"bor_chain_id"` // bor chain id
 
-	BorChainID string `json:"borChainID"` // bor chain id
+	AmqpURL           string `mapstructure:"amqp_url"`             // amqp url
+	HeimdallServerURL string `mapstructure:"heimdall_rest_server"` // heimdall server url
 
-	StakeManagerAddress  string `json:"stakeManagerAddress"`  // Stake manager address on main chain
-	RootchainAddress     string `json:"rootchainAddress"`     // Rootchain contract address on main chain
-	ValidatorSetAddress  string `json:"validatorSetAddress"`  // Validator Set contract address on bor chain
-	StateSenderAddress   string `json:"stateSenderAddress"`   // main
-	StateReceiverAddress string `json:"stateReceiverAddress"` // matic
+	StakeManagerAddress  string `mapstructure:"stakemanager_contract"`   // Stake manager address on main chain
+	RootchainAddress     string `mapstructure:"rootchain_contract"`      // Rootchain contract address on main chain
+	StateSenderAddress   string `mapstructure:"state_sender_contract"`   // main
+	StateReceiverAddress string `mapstructure:"state_receiver_contract"` // matic
+	ValidatorSetAddress  string `mapstructure:"validator_set_contract"`  // Validator Set contract address on bor chain
 
-	ChildBlockInterval uint64 `json:"childBlockInterval"` // Difference between header index of 2 child blocks submitted on main chain
+	ChildBlockInterval uint64 `mapstructure:"child_chain_block_interval"` // Difference between header index of 2 child blocks submitted on main chain
 
 	// config related to bridge
-	CheckpointerPollInterval int           `json:"checkpointerPollInterval"` // Poll interval for checkpointer service to send new checkpoints or missing ACK
-	SyncerPollInterval       int           `json:"syncerPollInterval"`       // Poll interval for syncher service to sync for changes on main chain
-	NoACKPollInterval        time.Duration `json:"noackPollInterval"`        // Poll interval for ack service to send no-ack in case of no checkpoints
+	CheckpointerPollInterval time.Duration `mapstructure:"checkpoint_poll_interval"` // Poll interval for checkpointer service to send new checkpoints or missing ACK
```

Comment:
Type Conflict.  In bridge checkpoint.go, 
startPolling(ctx context.Context, pollInterval int) 
expects int.